deny.toml

```diff
@@ -76,14 +76,14 @@ notice = "deny"
 ignore = [
   #"RUSTSEC-0000-0000",
 ]
-# Threshold for security vulnerabilities, any vulnerability with a CVSS score
+# Threshold for security vulnerabilities, any vulnerability with a CVSS scores
 # lower than the range specified will be ignored. Note that ignored advisories
 # will still output a note when they are encountered.
-# * None - CVSS Score 0.0
-# * Low - CVSS Score 0.1 - 3.9
-# * Medium - CVSS Score 4.0 - 6.9
-# * High - CVSS Score 7.0 - 8.9
-# * Critical - CVSS Score 9.0 - 10.0
```

Comment:
Haha not a bit deal but I think these changes are all from the find and replace, so might be good to revert the changes to this file